jina/peapods/peas/__init__.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, args: Union['argparse.Namespace', Dict]):
 
         # TODO: BAD pattern here!
         #   It is unnecessary to do "`a` in b and b.a" for `Namespace`, the only reason
-        #   for this check is because PyClient is using (abusing?) `BasePea` but PyCilent's args is not
+        #   for this check is because Client is using (abusing?) `BasePea` but PyCilent's args is not
```

Comment:
This was assuming that `PyClientRuntime` is of Pea type and inherits `BasePea`. With `runtime` refactoring, this was to be removed